latch_cli/services/local_dev.py

```diff
@@ -301,32 +346,49 @@ async def shell_session(
         loop,
     )
 
+    resize_event_queue = asyncio.Queue()
+
+    def new_sigwinch_handler(signum, frame):
+        if isinstance(old_sigwinch_handler, Callable):
```

Comment:
would handle the cases when [you dont get a callable](https://docs.python.org/3/library/signal.html#signal.getsignal)

Reply:
i also remember there is a builtin to do this - https://docs.python.org/3/library/functions.html#callable, but i actually can't think of why this is better

i'm pretty sure `isinstance(obj, Callable) iff obj has __call__` which is exactly what builtin does, literally cant find anything else about it

Reply:
the cases where i dont get a callable arent things i dont need to handle - SIG_IGN is literally ignore the signal in this process, SIG_DFL for the signal im handling here (SIGWINCH) means the current handler does the default action for SIGWINCH, which is also to ignore it (https://www.gnu.org/software/libc/manual/html_node/Miscellaneous-Signals.html), and None ofc means that there is no handler for it